DependencyInjection/Configuration.php

```diff
@@ -28,6 +30,21 @@ public function getConfigTreeBuilder()
             ->children()
                 ->scalarNode('sylius_base_url')->isRequired()->end()
                 ->booleanNode('auto_publish')->defaultFalse()->end()
+                ->arrayNode('default_taxon_adapter')->canBeEnabled()->end()
```

Comment:
would call this `enable_taxon_category_adapter`

Reply:
have added it like this:

```
sulu_sylius_consumer:
    default_taxon_adapter:
        enabled: true
```

so the name `enable_taxon_category_adapter` would not be enough. i will call it `taxon_category_adapter`